translations/README.mx_spanish.md

```diff
@@ -0,0 +1,210 @@
+  - [Buscar APIs](https://realtime-poll.demo.hasura.app/console)
+
+### Videos
+
+* [Agregar GraphQL a una instancia "auto-hosted" de GitLab](https://www.youtube.com/watch?v=a2AhxKqd82Q) (*3:44 mins*)
+* [Aplicación ToDo con Auth0 y servidor GraphQL](https://www.youtube.com/watch?v=15ITBYnccgc) (*4:00 mins*)
+* [GraphQL en GitLab integrado con aunteticación de GitLab](https://www.youtube.com/watch?v=m1ChRhRLq7o) (*4:05 mins*)
+* [Panel para 10 millones de viajes con geolocalización (PostGIS, Timescale)](https://www.youtube.com/watch?v=tsY573yyGWA) (*3:06 mins*)
+
+
+## Soporte y solución de problemas
+
+La documentación y la comunidad te ayudarán a solucionar la mayoría de los problemas. Si encuentras un error o necesitas contactarte con nosotros, puedes hacerlo usando uno de los siguientes canales: 
+
+* Soporte y retroalimentación: [Discord](https://discord.gg/vBPpJkS)
+* Problemas y seguimiento de errores: [GitHub issues](https://github.com/hasura/graphql-engine/issues)
+* Sigue las actualizaciones del producto: [@HasuraHQ](https://twitter.com/hasurahq)
+* Habla con nosotros en nuestro [chat](https://hasura.io)
+
+Estamos comprometidos a fomentar un ambiente abierto y acogedor en la comunidad. Por favor revisa el [Código de Conducta](../code-of-conduct.md).
+
+Si quieres reportar un problema de seguridad, por favor [lee esto](../SECURITY.md).
+
+## Contribuir
+
+Revisa nuestra [guía de contribución](../CONTRIBUTING.md) para más detalles.
+
+## Archivos de marca
+
+Los archivos de marca de Hasura (logotipos, la mascota de Hasura, la insignia: "powered by", etc.) pueden ser encontradas en la carpeta [assets/brand](../assets/brand). Siéntete libre de usarlos en tu
+aplicación, sitio web, etc. Estaremos encantados si agregas la insignia "Powered by Hasura" a tus aplicaciones creadas con Hasura. ❤️
+
+<div style="display: flex;">
+  <img src="assets/brand/powered_by_hasura_blue.svg" width="150px"/>
+  <img src="assets/brand/powered_by_hasura_white.svg" width="150px"/>
+</div>
+
+```html
+<!-- For light backgrounds -->
+<a href="https://hasura.io">
+  <img width="150px" src="https://graphql-engine-cdn.hasura.io/img/powered_by_hasura_blue.svg" />
+</a>
+
+<!-- For dark backgrounds -->
+<a href="https://hasura.io">
+  <img width="150px" src="https://graphql-engine-cdn.hasura.io/img/powered_by_hasura_white.svg" />
+</a>
+```
+
+## Licencia
+
+El núcleo del motor GraphQL está disponible bajo la [Licencia Apache 2.0](https://www.apache.org/licenses/LICENSE-2.0) (Apache-2.0).
+
+Todos **los contenidos** (excepto aquellos incluidos en los directorios [`server`](../server), [`cli`](../cli) y
+[`console`](../console)) están disponibles bajo la [Licencia MIT](../LICENSE-community).
+Esto incluye todo en los directorios [`docs`](../docs) y [`community`](../community).
+
+## Traducciones
+
+Este archivo está disponible en los siguientes idiomas:
+
+- [Japonés :jp:](../translations/README.japanese.md) (:pray: [@moksahero](https://github.com/moksahero))
```

Comment:
There are more translations here by now. Could you add them please?

Reply:
done